src/clientapi/native/launchClient.js

```diff
@@ -20,7 +20,7 @@ GLOBAL.console = logging.get('console');
 window.self = window;
 
 // add bluebird promise implementation to global scope
-import Promise;
+import ..api.Promise as Promise;
```

Comment:
is `api` no longer in the path? if so, it'll probably break a fair bit of code. if not, no need to change the import to a relative one?